src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLMoneyImpl.java

```diff
@@ -0,0 +1,24 @@
+package org.opentripplanner.ext.legacygraphqlapi.datafetchers;
+
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import java.util.Currency;
+import org.opentripplanner.ext.legacygraphqlapi.generated.LegacyGraphQLDataFetchers;
+import org.opentripplanner.transit.model.basic.Money;
+
+public class LegacyGraphQLMoneyImpl implements LegacyGraphQLDataFetchers.LegacyGraphQLMoney {
+
+  @Override
+  public DataFetcher<Integer> amount() {
+    return env -> getSource(env).cents();
```

Comment:
Do you mind changing the `org.opentripplanner.transit.model.basic.Mony` naming as well? cents -> amount

Reply:
Not at all. It's here: https://github.com/opentripplanner/OpenTripPlanner/pull/4979/commits/b5343467607c2270e387f2526e7574ca11f9f3f9